src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
++---
+title: Inclusive Making and Hacking
+layout: default
+category: Methods
+---
+
+Hackathons, makeathons and similar events have become very popular in recent years, both in traditional learning contexts such as schools, colleges and universities, and in broader world.
```

Comment:
What makes a "hackathon" and a "makeathon" similar? Is there an opportunity here to summarize just in a word or two what these kinds of events are about? Are they "community-oriented" events? Or "participatory"? What's common to them all?